### PR TITLE
add configuration file of pando chain

### PR DESCRIPTION
**Pull request by uniblok**
Add pando chain testnet network configuration file.

### CODE REVIEW

File: _data/chains/eip155-3601.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    "networkId": 3601,
+    "explorers": [
+      {
+        "name": "Pando Mainnet Explorer",
+        "url": "https://explorer.pandoproject.org/",
+        "standard": "EIP3091"
+      }
+    ]
+  }
+
```
Comment (ligi) [inline _data/chains/eip155-3601.json:23]:
```suggestion
```

File: _data/chains/eip155-3601.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+      "decimals": 18
+    },
+    "infoURL": "https://www.pandoproject.org/",
+    "shortName": "pando-mainnet",
+    "chainId": 3601,
+    "networkId": 3601,
+    "explorers": [
+      {
+        "name": "Pando Mainnet Explorer",
+        "url": "https://explorer.pandoproject.org/",
```
Comment (ligi) [inline _data/chains/eip155-3601.json:18]:
```suggestion
        "url": "https://explorer.pandoproject.org",
```

**Review by ligi — CHANGES_REQUESTED** [review]

**Comment by uniblok** [comment]
Thanks for your consideration, we will update the json file and raise a
pull request by next week.

**Pull request closed without merging**